snapcraft/script/bridge.stop

```diff
@@ -3,6 +3,8 @@
 ip addr flush dev conjureup0
 ip link set dev conjureup0 down
 
+sleep 5
+
```

Comment:
seems error prone, is there any thing we can loop on to be sure we've waited long enough?